python/ray/autoscaler/_private/fake_docker/example.yaml

```diff
@@ -0,0 +1,64 @@
+# Example command to start a cluster with this config:
+#
+# RAY_FAKE_DOCKER=1 ray start --autoscaling-config=example.yaml --head --block
+#
+# Alternatively, you can programmatically create a fake autoscaling cluster
+# using ray.cluster_utils.AutoscalingCluster.
+cluster_name: fake_docker
+max_workers: 8
+provider:
+    type: fake_docker
+    project_name: fake_docker
+    image: rayproject/ray:nightly
+    shared_volume_dir: /tmp/fake_docker
+    head_resources:
+      CPU: 4
+      GPU: 0
```

Comment:
this is a bit awkward but basically the node provider on the head node does not have access to the rest of the config out of the box

Reply:
We can change the API to pass the config I think

Reply:
I've left this out for now. We can do the API change in a follow-up PR. I'd like to leave it our here as it would touch too many files and it would be good to have a separate commit for that.